opensearchpy/_async/client/__init__.py

```diff
@@ -224,6 +224,7 @@ class as kwargs, or a string in the format of ``host[:port]`` which will be
         self.ingest = IngestClient(self)
         self.nodes = NodesClient(self)
         self.remote = RemoteClient(self)
+        self.search_pipeline = SearchPipelineClient(self)
```

Comment:
@ArbitraryYu, thanks for your contribution. Just a few adjustments needed. 

- No modification needed for this file. In a recent PR, `self.search_pipeline = SearchPipelineClient(self)` was added.